chocolatey-community/chocolatey-community-validation
# (#32) Update Help URLs to short links

## Description Of Changes

This updates all the URLs shown when a rule fais the validation to use
ch0.co shortlink instead of the full URL to the documentation site.

## Motivation and Context

Using a full URL when outputting in the console uses unnecessary screenspace, and using a short link allows us more customizability if there is a need for it in the future.

## Testing

1. Run `choco new default`.
2. With the extension installed, run `choco pack <PATH_TO_NUSPEC>` with the path to the created package.
3. Verify all URLs outputted in the console uses the `ch0.co` shortlink.

### Operating Systems Testing

- Windows 10

## Change Types Made


## Change Checklist


## Related Issue

Fixes #32
